style.css

```diff
@@ -1,3 +1,64 @@
-.header {
-    color: blue;
-}
\ No newline at end of file
+body {
+  margin: 0;
+  padding: 0;
+  display: block;
+  background: #fff;
+  font-family: 'Inter', sans-serif;
+}
+
+header {
+  padding: 0 35px;
+  display: flex;
+  justify-content: right;
+  align-items: right;
+  min-width: 375px;
+}
+
+h1 {
+  font-style: normal;
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  text-align: center;
+  margin-top: 1.5em;
+  margin-left: 27px;
+}
+
+h1 span {
+  color: #36b37f;
+}
+
+.headline {
+  background: url(./images/headline-background.svg) no-repeat;
+  display: flex;
+  flex-direction: column;
+  background-size: 100% 100%;
+  align-items: center;
+  text-align: center;
+}
```

Comment:
- [ ] As pointed out earlier, kindly ensure that your design is a close replica of the given template. 👍 As developers, it's always good that we pay attention to details; recruiters out there watch out for this. ℹ️ 

> |     | Your Design                                  | Figma Design                                  |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     | <img width="282" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235100218-59eadf4e-f79e-44d6-82e3-18c2f82103e7.png"> | <img width="227" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235099869-acb4bf25-e496-4361-a878-027210826f4d.png"> |